games-action/postal/patches/postal-1.0.0.patchset

```diff
@@ -0,0 +1,401 @@
+From 08422e8fd073291e3acdb0f15751e2d68501fcc5 Mon Sep 17 00:00:00 2001
+From: Craig Watson <watsoncraigjohn@gmail.com>
+Date: Mon, 30 Aug 2021 12:57:18 +0100
+Subject: Haiku support
+
+
+diff --git a/CompileOptions.h b/CompileOptions.h
+index 57b1799..e5dae0f 100644
+--- a/CompileOptions.h
++++ b/CompileOptions.h
+@@ -360,7 +360,7 @@
+ 				#endif
+ 			#endif
+ 		#else
+-			#define PREFS_FILE	"POSTAL.INI"
++			#define PREFS_FILE	"/boot/system/settings/POSTAL.INI"
```

Comment:
Preferences are usually in /boot/home/config/settings

Reply:
This seems to be done in other packages, including games, but I have changed it anyway